samples/CosmosDB/cosmos-db-vnet-filter/azurepolicy.rules.json

```diff
@@ -0,0 +1,25 @@
+{
+	"if": {
+		"allOf": [
+			{
+				"field": "type",
+				"equals": "Microsoft.DocumentDB/databaseAccounts"
+			},
+			{
+				"anyOf": [
+					{
```

Comment:
>					 [](start = 0, length = 5)

nit: convert tabs to spaces, please
(everywhere)

Reply:
Done!

Reply:
Thanks!